contracts/Octobay.sol

```diff
@@ -210,14 +210,8 @@ contract Octobay is Ownable, ChainlinkClient, BaseRelayRecipient {
     function setGovTokenForIssue(string calldata _issueId, OctobayGovToken _govToken) public {
         // Ensure they're giving us a valid gov token
         require(bytes(octobayGovernor.projectsByToken(_govToken)).length != 0, "Invalid _govToken");
-        bool hasPermission = false;
-        uint256 govNFTId = octobayGovNFT.getTokenIDForUserByGovToken(msg.sender, _govToken);
-        if (govNFTId != 0 && octobayGovNFT.hasPermission(govNFTId, OctobayGovNFT.Permission.SET_ISSUE_GOVTOKEN)) {
-            hasPermission = true;
-        } else {
-            // Do other permission checks here, e.g. oracle calls
```

Comment:
yes I think we can really just rely on the NFTs. That was actually the main motivation, to reduce the number of oracle calls.